src/aat/resources/features/F-050/F-050.feature

```diff
@@ -0,0 +1,57 @@
+@F-050
+Feature: Validate a set of fields as Case worker
+
+  Background: Load test data for the scenario
+    Given an appropriate test context as detailed in the test data source
+
+  @S-301
+  Scenario: must validate when all fields are valid
+    Given a user with [a detailed profile in CCD]
+    When a request is prepared with appropriate values
+    And it is submitted to call the [validation of a set of fields as Case worker] operation of [CCD Data Store]
+    Then a positive response is received
+    And the response [has the 200 return code]
+    And the response has all other details as expected
+
+  @S-298
+  Scenario: must not validate when field validation fails
+    Given a user with [a detailed profile in CCD]
+    When a request is prepared with appropriate values
+    And the request [uses a value that exceeds the field max limit]
+    And it is submitted to call the [validation of a set of fields as Case worker] operation of [CCD Data Store]
+    Then a negative response is received
+    And the response [has the 422 return code]
+    And the response has all other details as expected
+
+  @S-300
+  Scenario: must return 403 when request provides authentic credentials without authorised access to the operation
+    Given a user with [a detailed profile in CCD]
```

Comment:
"an active profile in CCD" would be better

Reply:
Updated:  Given a user with [an active profile in CCD]